### PR TITLE
Added lit docs

### PR DESCRIPTION
**Pull request by emilbonnek**
@ai 

I added a section in the readme for the lit integration we have been talking about in #95. It only covers the most simple way to use it with the reactive controllers directly and I just used the example from the Solid integration above.

However, I don't think we should merge this just yet. The code is not published to npm (I don't think I can publish to the nanostores npm space?) and the lit integration repository is still private in the nanostores GitHub organization.

### CODE REVIEW

**Comment by ai** [comment]
Looks good. Do you want to publish and make repo public?

**Comment by emilbonnek** [comment]
I think I will need to be added to the npm organization before I can publish?

File: README.md [hunk 1]
```diff
@@ -522,6 +522,32 @@ export function Header({ postId }) {
...
 
+### Lit
+
+Use [`@nanostores/lit`] and `StoreController` reactive controller
+to get store’s value and re-render component on store’s changes.
+
+```ts
+import { StoreController } from '@nanostores/lit';
+import { profile } from '../stores/profile.js'
+import { Post } from '../stores/post.js'
```
Comment (euaaaio) [inline README.md:533]: Oops, you missed a few semicolons. But in fact, we don't use them at all in the docs 😅

**Comment by ai** [comment]
@emilbonnek sure! What is npm user name?

**Comment by emilbonnek** [comment]
Same as on here "emilbonnek"

**Comment by ai** [comment]
Done. I sent you an invitation. Ping when you accept it and release the first version of `@nanostores/lit`.

File: README.md (hunk 1 above)
Comment (emilbonnek) [reply README.md:533]: That is indeed an oopsie 😅 Thanks

**Comment by emilbonnek** [comment]
@ai I accepted the invitation and published it now.

**Comment by ai** [comment]
Great! Are we ready to accept PR?

**Comment by emilbonnek** [comment]
I think you can accept it now.

**Pull request merged**